components/infobox/wikis/rainbowsix/infobox_person_player_custom.lua

```diff
@@ -0,0 +1,248 @@
+		player.args.team2 = PlayerTeamAuto._main{team = 'team2'}
+	end
+
+	player.args.history = Template.safeExpand(frame, 'TeamHistoryAuto', {player = player.args.id})
+
+	player.adjustLPDB = CustomPlayer.adjustLPDB
+	player.createBottomContent = CustomPlayer.createBottomContent
+	player.createWidgetInjector = CustomPlayer.createWidgetInjector
+	player.defineCustomPageVariables = CustomPlayer.defineCustomPageVariables
+
+	_args = player.args
+
+	return player:createInfobox(frame)
+end
+
+function CustomInjector:parse(id, widgets)
+	if id == 'status' then
+		return {
+			Cell{name = 'Status', content = CustomPlayer._getStatusContents()},
+			Cell{name = 'Years Active (Player)', content = {_args.years_active}},
+			Cell{name = 'Years Active (Org)', content = {_args.years_active_manage}},
+			Cell{name = 'Years Active (Coach)', content = {_args.years_active_coach}},
+			Cell{name = 'Years Active (Talent)', content = {_args.years_active_talent}},
+			Cell{name = 'Time Banned', content = {_args.time_banned}},
+		}
+	elseif id == 'role' then
+		return {
+			Cell{name = 'Role', content = {
+				CustomPlayer._createRole('role', _args.role),
+				CustomPlayer._createRole('role2', _args.role2)
+			}},
+		}
+	elseif id == 'history' then
+		table.insert(widgets, Cell{
+			name = 'Retired',
+			content = {_args.retired}
+		})
+	end
+	return widgets
+end
+
+function CustomInjector:addCustomCells(widgets)
+	-- Signature Operators
+	table.insert(widgets,
+		Builder{
+			builder = function()
+				local operatorIcons = Array.map(Player:getAllArgsForBase(_args, 'operator'),
+					function(operator, _)
+						return OperatorIcon.getImage{operator, size = _SIZE_OPERATOR}
+					end
+				)
+				return {
+					Cell{
+						name = #operatorIcons > 1 and 'Signature Operators' or 'Signature Operator',
+						content = {
+							table.concat(operatorIcons, '&nbsp;')
+						}
+					}
+				}
+			end
+		})
+	-- Active in Games
+	table.insert(widgets,
+		Builder{
+			builder = function()
+				local activeInGames = {}
+				Table.iter.forEachPair(_GAMES,
+					function(key)
+						if _args[key] then
+							table.insert(activeInGames, _GAMES[key])
+						end
+					end
+				)
+				return {
+					Cell{
+						name = #activeInGames > 1 and 'Games' or 'Game',
+						content = {
+							table.concat(activeInGames, '<br>')
+						}
+					}
+				}
+			end
+		})
+	return widgets
+end
+
+function CustomPlayer:createWidgetInjector()
+	return CustomInjector()
+end
+
+function CustomPlayer:adjustLPDB(lpdbData)
+	lpdbData.extradata.role = Variables.varDefault('role')
+	lpdbData.extradata.role2 = Variables.varDefault('role2')
+
+	lpdbData.extradata.signatureOperator1 = _args.operator1 or _args.operator
+	lpdbData.extradata.signatureOperator2 = _args.operator2
+	lpdbData.extradata.signatureOperator3 = _args.operator3
+	lpdbData.extradata.signatureOperator4 = _args.operator4
+	lpdbData.extradata.signatureOperator5 = _args.operator5
+	lpdbData.extradata.isplayer = Variables.varDefault('isplayer')
```

Comment:
we probably can move that to commons?!

Reply:
https://liquipedia.net/commons/Module:Portal_players should probably be updated instead to use `type`.

Reply:
alright

Reply:
Updated portal players to handle `type` as well.